ai-service/src/ai_service/db.py

```
@@ -45,3 +45,35 @@ def add_chunks(chunks: list[str], embeddings: list[list[float]]) -> None:
+        text_embedding: Vector embedding of a user query.
+        number_of_results: Number of results to return (1-100).
+
+    Returns:
+        Dict containing 'documents', 'distances', 'metadatas', and 'ids'.
+
+    Raises:
+        DatabaseError: If the query fails.
+        InvalidParam: If parameters are invalid.
+    """
+    if text_embedding is None or len(text_embedding) == 0:
+        raise InvalidParam("Query embedding is empty.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)